draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2254,7 +2267,7 @@ uCDN-A                         dCDN-B                         dCDN-C
         <vspace blankLines="1"/>
    Value: Internet date and time as defined in <xref target="RFC3339"/>.
         <vspace blankLines="1"/>
-        Mandatory: No. but at least one of "start" or "end" MUST be present and non-empty.
+        Mandatory: No. but at least one of the "start" or "end" MUST be present and non-empty.
```

Comment:
Adding "the" sounds more awkward here.  We're referring to one of these choices.  Maybe "either" is is a better choice than "the".